utam-compiler/src/test/resources/compose/apply/customListWithChain.json

```diff
@@ -0,0 +1,32 @@
+{
+  "elements": [
+    {
+      "name": "custom",
+      "type": "org-lwr/pageObjects/custom",
+      "selector": {
+        "css": ".css",
+        "returnAll" : "true"
```

Comment:
Quick nitpick but it should be a boolean here

Reply:
done